spanner/spanner_samples.rb

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def create_instance project_id:, instance_id:
+def create_instance project_id, instance_id, instance_config_id = "regional-us-central1"
```

Comment:
Please don't change from keyword arguments to positional arguments. (The usage change will cause the existing tests to fail, among other things.) You can set default values for a keyword argument:

```ruby
def create_instance project_id:, instance_id:, instance_config_id: "regional-us-central1"
```